kt/plugins/godot-gradle-plugin/build.gradle.kts

```diff
@@ -2,34 +2,25 @@ import godot.dependencies.gradle.DependenciesVersions
 
 plugins {
     kotlin("jvm")
-    `maven-publish`
     `java-gradle-plugin`
-    id("com.gradle.plugin-publish") version "0.14.0"
+    id("com.gradle.plugin-publish") version "1.1.0"
     id("com.utopia-rise.godot-publish")
     id("com.utopia-rise.godot-dependencies")
 }
 
 gradlePlugin {
+    website.set("https://github.com/utopia-rise/godot-kotlin-jvm")
+    vcsUrl.set("https://github.com/utopia-rise/godot-kotlin-jvm.git")
+
     plugins {
         create("godotPlugin") {
             id = "com.utopia-rise.godot-kotlin-jvm"
             displayName = "Gradle plugin for godot-kotlin-jvm"
             implementationClass = "godot.gradle.GodotPlugin"
             description = "Automatically setup of godot kotlin jvm specific project configurations"
-        }
-    }
-    isAutomatedPublishing = false
-}
-
-pluginBundle {
-    website = "https://github.com/utopia-rise/godot-kotlin-jvm"
-    vcsUrl = "https://github.com/utopia-rise/godot-kotlin-jvm.git"
-    tags = listOf("kotlin", "godot", "gamedev")
 
-    mavenCoordinates {
```

Comment:
Where do we set maven coordinates now ?

Reply:
The gradle publishing plugin does that automatically now for the gradle plugin portal release as far as i understood. But no way for me to test that yet without a release. But i think it's working correctly because for the publishToMavenLocal task i get the warning that those two publications will override each other. If the coordinates would not match, they wouldn't do that.

For our manual maven central release we define that still through the regular publishing block below

Reply:
In any case there seems to be no way anymore to define that manually as the `isAutomatedPublishing` flag (which was required to configure the coordinates manually) cannot be set to false anymore